all_test.go

```diff
@@ -154,24 +166,24 @@ func isPrefix(prefix, str string) bool {
 }
 
 func TestIdentityCaseSensitivity(t *testing.T) {
+	t.Log("Testing case sensitivity")
 	c := setup1(t)
 
-	if e := c.CreateAuthenticatorIdentity("JOE", "123"); e == nil || !isPrefix(ErrIdentityExists.Error(), e.Error()) {
+	if e := c.CreateUserStoreIdentity("JOE", "JOEusername", "JOEfirstname", "JOElastname", "JOE084", "123"); e == nil || !isPrefix(ErrIdentityExists.Error(), e.Error()) {
 		t.Errorf("CreateIdentity should fail because identities are case-insensitive. Instead, error is %v", e)
 	}
 	perm := Permit{}
 	roles := [2]byte{99}
 	perm.Roles = roles[:]
-	if e := c.SetPermit("JOE", &perm); e != nil {
+	if e := c.SetPermit(joeUserId, &perm); e != nil {
```

Comment:
This part of the test, that checks case sensitivity of Permits, is no longer valid, since you're now passing those APIs integer IDs.